src/policy/mod.rs

```diff
@@ -0,0 +1,44 @@
+// Rust Bitcoin Library
+// Written in 2014 by
+//   Andrew Poelstra <apoelstra@wpsoftware.net>
+//
+// To the extent possible under law, the author(s) have dedicated all
+// copyright and related and neighboring rights to this software to
+// the public domain worldwide. This software is distributed without
+// any warranty.
+//
+// You should have received a copy of the CC0 Public Domain Dedication
+// along with this software.
+// If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+//
+
+//! Policy
+//!
+//! This module exposes some constants and functions used in the reference implementation and which
+//! as a consequence defines some network rules.
+//!
+//! # *Warning*
+//! While the constants present in this module are very unlikely to change, they do not define
+//! Bitcoin. As such they must not be relied upon as if they were consensus rules.
```

Comment:
Maybe a link to the core code would be nice here and the bitcoind version from which the constants were taken?

Reply:
Added the version and the tip